model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -0,0 +1,92 @@
+import sys
+import inspect
+
+from torch import utils, nn
+from torchvision import transforms
+
+from dffml.base import BaseDataFlowFacilitatorObject
+from dffml.util.entrypoint import base_entry_point, entrypoint
+from .config import make_pytorch_config
+
+
+def create_layer(layer_dict):
+    try:
+        sequential_dict = nn.Sequential()
+        for name, layer in layer_dict.items():
+            parameters = {k: v for k, v in layer.items()}
+            layer_name = parameters.pop("name")
+
+            sequential_dict.add_module(
+                name, getattr(nn, layer_name)(**parameters)
+            )
+        return sequential_dict
+    except AttributeError:
+        parameters = {k: v for k, v in layer_dict.items()}
+        layer_name = parameters.pop("name")
+        return getattr(nn, layer_name)(**parameters)
+
+
+class NumpyToTensor(utils.data.Dataset):
+    def __init__(self, data, target=None, size=224):
+        self.data = data
+        self.target = target
+        self.size = size
+
+    def transform(self, data, size):
+        return transforms.Compose(
+            [
+                transforms.ToPILImage(),
+                transforms.Resize(size),
+                transforms.CenterCrop((size, size)),
+                transforms.ToTensor(),
+                transforms.Normalize(
+                    [0.485, 0.456, 0.406], [0.229, 0.224, 0.225]
```

Comment:
Let's add List[float] options for the mean and std parameters to [Normalize](https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.Normalize) within `PyTorchModelConfig` do not append Normalize to the list passed to Compose if either mean or std is None.